File: headers/analyze.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Analyze
{
	public static class Analyze
	{
		public static void AddMarker(char* func_name, unsigned int g)
		{
			FileStream fstream =  new FileStream("output.txt", FileMode.OpenOrCreate);
			fstream.Write(func_name,)
		}
	}
}

File: headers/analyze.c
//
// Created by delvin on 16.11.2019.
//
#include "analyze.h"
#include <stdio.h>
unsigned int callgrind_checker = 0;

void add_sensor(char* func_name, unsigned int callgrind_checker)
{
    static unsigned int id = 0;
    if(callgrind_checker==0)
        output = fopen("output.txt", "w");
    for(int i = 0; i < callgrind_checker; i++)
        fprintf(output,"%s", "#");
    fprintf(output, "id%d: %s\n", id++, func_name);
}
